test/ne.jl

```diff
@@ -10,21 +10,39 @@ function check_pressure_status_ne(sol)
     end
 end
 
-
-#Check the qp gas and power flow model, this is contrived to make sure something is built on both sides
-
+# Check the qp gas and power flow model. This is contrived to make sure
+# something is built on both sides.
 @testset "test qp ne" begin
-    @testset "IEEE 14 Belgian NE case" begin
-        normalization = 1e-8
-        gas_ne_weight = 1.0
-        power_ne_weight = 1.0
+    @testset "IEEE 14 and Belgian Network Expansion" begin
+        normalization = 1.0e-8
+        gas_ne_weight, power_ne_weight = [1.0, 1.0]
+        gfile, pfile = ["../test/data/matgas/belgian_ne.m", "../test/data/case14-ne.m"]
```

Comment:
Same note as in the code.  Did you try instead,
```
gfile, pfile = "../test/data/matgas/belgian_ne.m", "../test/data/case14-ne.m"
```

Reply:
Don't know why I was putting these in an array... Anyway, the reason for these sorts of lines is that I prefer longer one-liners than multiple short and similar lines.